packages/vaadin-text-field/test/text-area.test.js

```diff
@@ -340,3 +340,49 @@ import '../vaadin-text-area.js';
     });
   });
 });
+
+describe('helper text', () => {
+  describe('slotted', () => {
+    let field, helper;
+
+    beforeEach(() => {
+      field = fixtureSync(`
+        <vaadin-text-area label="outer">
+          <vaadin-text-area label="inner" slot="helper">
+            <vaadin-text-area label="inner" slot="helper"></vaadin-text-area>
+          </vaadin-text-area>
+        </vaadin-text-area>
+      `);
+      helper = field.querySelector('[slot="helper"]');
+      field.focus();
+    });
+
+    it('should get focus when clicked', () => {
+      expect(field.hasAttribute('focused')).to.be.true;
+    });
+
+    it('helper should get focus when clicked', () => {
+      helper.inputElement.focus();
+      expect(field.hasAttribute('focused')).to.be.false;
+      expect(helper.hasAttribute('focused')).to.be.true;
+    });
```

Comment:
These tests need to be updated and aligned with other tests added in this PR:

1. The first test can be removed as we do not need to test `focused` attribute here.
2. The second test needs to use `click()` and not `focus()`, and the spy as in the `vaadin-custom-field` test.